pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -189,19 +190,18 @@ func (r *reconciler) add(rule *CompletedRule) error {
 	if rule.Direction == v1beta1.DirectionIn {
 		// Addresses got from source Pod IPs.
 		from1 := podsToOFAddresses(rule.FromAddresses)
-		// Addresses and ExceptAddresses got from IPBlocks.
-		from2, exceptFrom := ipBlocksToOFAddresses(rule.From.IPBlocks)
+		// Diff Addresses between From and Except of IPBlocks.
```

Comment:
Do you mean "Diff of addresses"? As in "Addresses in From which are not in Except"?